sap-xmake-build.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash -l
+echo "Inside file -- sap-xmake-build.sh --"
+
+# find this script and establish base directory
+curr_dir=`dirname $0`
+MY_DIR=`cd $curr_dir; pwd`
+echo "[INFO] Executing in ${MY_DIR}"
+
+PACKAGE_NAME=${PACKAGE_NAME:-"scala-pickling"}
+PACKAGE_BRANCH=${PACKAGE_BRANCH:-"0.11.x"}
+BUILD_BRANCH=${BUILD_BRANCH:-"v0.11.x-M1_2.11"}
+ALTISCALE_RELEASE="${ALTISCALE_RELEASE:-5.0.0}"
+
+mkdir -p hack_target
+pushd hack_target
+rm -rf *.jar
+popd
+
+rm -rf install-dir-*
+rm -rf rpm-dir-*
+
+echo "ok - $(whoami) user is going to build the project"
+echo "ok - fpm is located at $(which fpm)"
+echo "ok - PATH=$PATH"
+echo "ok - BUILD_BRANCH=$BUILD_BRANCH"
+echo "ok - PACKAGE_BRANCH=$PACKAGE_BRANCH"
+
+ret=0
+export DATE_STRING=${DATE_STRING:-$(date -u +%Y%m%d%H%M)}
+RPM_INSTALL_DIR=${MY_DIR}/picklingrpmbuild
+echo "RPM_INSTALL_DIR : $RPM_INSTALL_DIR"
+pushd ${MY_DIR}
+
+# pushd ${WORKSPACE}/pickling
+# sbt package
+# ret=$?
+
```

Comment:
Since ${WORKSPACE} is part of the old file and is not defined in this file, we can probably delete lines 34-36 else it might be confusing

Reply:
+1 ... to remove old comments